docs/modules/record-encryption/proc-configuring-record-encryption-filter.adoc

```diff
@@ -35,14 +35,21 @@ filters:
       selector: <KEK_selector_service_name> # <3>
       selectorConfig:
         template: "KEK_${topicName}" # <4>
+      experimental:
+        "encryptionDekRefreshAfterWriteSeconds": 3600 # <6>
```

Comment:
why quote the keys?

Reply:
I must have had some fever dream that you could have different key types and this showed the keys are Strings, will remove em.